Meta/run.py

```diff
@@ -0,0 +1,906 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2018-2023, the SerenityOS developers.
+# Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+from __future__ import annotations
+
+import os
+import re
+import sys
+from dataclasses import dataclass, field
+from enum import Enum, unique
+from itertools import chain, repeat
+from os import access, environ
+from pathlib import Path
+from shutil import which
+from subprocess import run
+from typing import Any, Callable, Literal
+
+QEMU_MINIMUM_REQUIRED_MAJOR_VERSION = 6
+QEMU_MINIMUM_REQUIRED_MINOR_VERSION = 2
+
+
+class RunError(Exception):
+    pass
+
+
+@unique
+class Arch(Enum):
+    """SerenityOS architecture, not host architecture."""
+
+    Aarch64 = "aarch64"
+    x86_64 = "x86_64"
+
+
+@unique
+class QEMUKind(Enum):
+    """VM distrinctions determining which virtualizers *might* be used."""
```

Comment:
distrinctions?

Reply:
updated the comment to be more accurate, and fix typo